.github/workflows/ci.yml

```diff
@@ -179,7 +179,11 @@ jobs:
     - name: Evaluate test results vs Linux file
       run: |
         python ${SAMNTDIR}/test/compare_lk_test_output.py ${SAMNTDIR}/test_results_linux64.csv ${SAMNTDIR}/build/test_results_linux64.csv false
-        
+    
```

Comment:
@tyneises , @taylorbrown75  this should address your workflow!

@brtietz  thanks for restoring